config/credentials.yml.enc

```diff
@@ -1 +1 @@
-9zcbF5GBooH8TtamG4FmW/CNEYbOhC6xX08XEE4qV+cB9VYZD5j9tWjlWPg7H6+ccNP3N6ssrn77ATaPx4D4SsZQMDAmuQ8RAemMBMPPQy2e5n+Lj306gqXO65bN/Aez8TJxmU4f1ySInXBa0SUZk7RKrf8w53I0OWywGqdUcuoH6lY61e39MLvaIKkepy3F9SR/Nhey3+PFyfd64W/yQQqxZjF4Y+hH1IHGavgsx2GcOnmlqUNoJfnnCA9EaiB0jaOahAV7rll6i2cVikT6CZy10RRJB3EGgrAO33zrlPTGXdQgpDSSz5z7ZgmOrFDeQrs5qOU8teJlpAdh/dFpXmwj+fcOltVQa/9yk+4HB/kx/ZtPygrjxK1T6dzRRQYl7TRpHv29q3Ie8I61joH/LtN4JuE9xMXwL0fD--X1by2OXQb4BJ5D72--EscjIvB0+TvVd56hdqcdaA==
\ No newline at end of file
+/5f3HrPXuf9uZJx8DyNv7qx77yMUW90zldguMtNWWFh0d8chXHQhJ+E28kW+QrFXZpuq1d8Sy8GW+Uze1+EfLTcxMrUdGIMIYQj0slPy8ZBijX4TU7cYnDeAzPh6Lg5DSYjDC/7GqSGH77srk2648OHh98in9TzKfZagwFbupghKx3NblfWW91rkO5la1Up6ZKpIaHlaWDZ7LjGc256LAqYnPnyXbCvE+uVPPpsov2DAuJX3YNjfGVPUVVzDOqZgGp3D4IHFwS0U3hASuMr8WSDWGu7w6m/JAWMu4OvbCnJZeMtwVv9n6kAuIKnYXLcb5NJ7FlvMuuQ61AqMPwPG98fzpKVlBiNq2rCk38ondQOqZ78BgXpDLf43eRJi4jEtgfr2TCvulw2o4/JvaLVPl8v0WGuVQLV2qQ7k--i5b97mMNgPTFNj7Y--HEpoduciEb64wc44lPy8FQ==
```

Comment:
Are you sure this can be public? Might need to be in .gitignore

Reply:
This can be public. This is an encrypted value generated with the file `config/master.key` which is alrealdy ignored in `.gitignore`.